src/users/user.service.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@nestjs/common';
+import { User } from './user.entity';
+
+@Injectable()
+export class UserService {
+  async findAll(): Promise<User[]> {
+    return User.find();
```

Comment:
Better to use repositories for all work with Database.

https://docs.nestjs.com/techniques/database#repository-pattern
https://ru.wikipedia.org/wiki/%D0%A0%D0%B0%D0%B7%D0%B4%D0%B5%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5_%D0%BE%D1%82%D0%B2%D0%B5%D1%82%D1%81%D1%82%D0%B2%D0%B5%D0%BD%D0%BD%D0%BE%D1%81%D1%82%D0%B8

Reply:
@haltaction fixed